app/src/main/java/com/asfoundation/wallet/analytics/RakamEventLogger.kt

```diff
@@ -0,0 +1,55 @@
+package com.asfoundation.wallet.analytics
+
+import android.util.Log
+import cm.aptoide.analytics.AnalyticsManager
+import cm.aptoide.analytics.EventLogger
+import io.rakam.api.Rakam
+import org.json.JSONException
+import org.json.JSONObject
+
+class RakamEventLogger : EventLogger {
+
+  companion object {
+    const val TAG = "RakamEventLogger"
+  }
+
+  override fun setup() {
```

Comment:
Since we need to implement this method it's better to make it return Unit:

  override fun setup() = Unit